### PR TITLE
added tecplot easyconfig

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
@boegel good to go?

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/t/tecplot360ex/tecplot360ex.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+installername = '%(name)s_%(version)s.sh'
+sources = [installername]
+
+sanity_check_paths = {
+    'files': ['bin/tec360'],
+    'dirs': []
+}
+install_cmd = 'chmod +x %s && ./%s --skip-license --prefix=%%(installdir)s' % (installername, installername)
+moduleclass = 'vis'
+
```
Comment (boegel) [inline easybuild/easyconfigs/t/tecplot360ex/tecplot360ex.eb]: style fix pls

Comment (boegel) [inline easybuild/easyconfigs/t/tecplot360ex/tecplot360ex.eb]: i.e. blank line above moduleclass, no trailing blank line at the end

**Comment by boegel** [comment]
You now have two easyconfig files? Doesn't look right...

**Comment by JensTimmerman** [comment]
@boegel test report?

File: easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+easyblock = 'Binary'
+
+name = 'tecplot360ex'
```
Comment (boegel) [inline easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb]: can't find any reference to the 'ex' part?

Comment (boegel) [inline easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb]: ah, wait, here it is

Comment (boegel) [inline easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb]: http://www.tecplot.com/products/tecplot-360/

File: easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+version = 'linux64'
+
+homepage = 'http://www.tecplot.com/'
```
Comment (boegel) [inline easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb]: change to http://www.tecplot.com/products/tecplot-360/ ?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f5784682012b25827728 for a full test report.

File: easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb (hunk 3 above)
Comment (JensTimmerman) [inline easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb]: tecplot-360 ex is actually something different from tecplot-360 I believe, but the website is very unclear...

**Comment by boegel** [comment]
OK, going in, thanks @JensTimmerman!

**Pull request merged**